waspc/README.md

```diff
@@ -172,33 +166,34 @@ On any changes you do to the source code of Wasp, Wasp project gets recompiled,
 
 
 ## Building / development (detailed)
-Some useful stack commands:
-- `stack build` to build the project, including `wasp` binary which is both CLI and compiler in one.
-- `stack exec wasp-cli <arguments>` to run the `wasp` binary that you have built.
-- `stack test` to build the whole project + tests and then also run tests.
-- `stack build --file-watch` -> live watch, reruns every time a file changes. But we prefer using `ghcid`, it is faster.
-- `stack build --pedantic` -> sets -Wall and -Werror ghc options.
-- `stack build --profile`
-- `stack build --trace`
-- `stack install` -> builds the project and places the binary so it is in PATH (and you can call it directly with `wasp`).
-- `stack ghci` -> opens ghci in the context of the project, allowing you to load and run local modules.
-- `stack clear` -> clear all generated files/artifacts.
-
-For live compilation and error checking of your code we recommend using `ghcid`.
-You can install it globally with `stack install ghcid` and then just type `ghcid --command=stack ghci` when in the project -> it will watch for any file changes and report errors.
+Some useful cabal commands:
+- `cabal update` to update your package information.
+- `cabal build` to build the project, including `wasp` binary which is both CLI and compiler in one.
+- `cabal run wasp-cli <arguments>` to run the `wasp` binary that you have built.
+- `cabal test` to build the whole project + tests and then also run tests.
+- `cabal install` -> builds the project and places the binary so it is in PATH (and you can call it directly with `wasp`).
```

Comment:
Ok great, I did all of these changes except for removing `so it is`, because it has meaningful meaning (:D) -> binary isn't put in the PATH itself, it is put in place that is then available through PATH. I don't know which place but I know it will be accessible through PATH. So I would leave that part with `so it is`.